yarn.lock

```diff
@@ -7350,11 +7345,11 @@ mkdirp-promise@^5.0.1:
     mkdirp "*"
 
 mkdirp@*, mkdirp@^0.5.0, mkdirp@^0.5.1:
```

Comment:
ATTENTION: I have manually touched `yarn.lock` and resolve `mkdirp@*` to `mkdirp@0.5.3` because of https://github.com/ahmadnassri/mkdirp-promise/pull/88.

And likely because of https://classic.yarnpkg.com/en/docs/selective-version-resolutions#toc-limitations-Caveats, adding `resolutions: "mkdirp-promise/**/mkdirp": "^0.5.3"` to `package.json` does not work on my local builds.

🤯